test/Microsoft.AspNetCore.Razor.Runtime.Test/Microsoft.AspNetCore.Razor.Runtime.Test.csproj

```diff
@@ -29,4 +29,8 @@
     <PackageReference Include="xunit.runner.visualstudio" Version="$(XunitVersion)" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)'=='net461'">
```

Comment:
Was trying to keep the change as scoped as possible. We're still discussing internally if Razor should move to ns2.0. If it does, then this will move to common.props.